spec/features/viewing_peeps_spec.rb

```diff
@@ -0,0 +1,29 @@
+feature 'Viewing peeps' do
+  scenario 'peeps appear in reverse chronological order' do
+    first_peep = Peep.add(content: 'First peep!')
+    second_peep = Peep.add(content: 'Second peep!')
+    third_peep = Peep.add(content: 'Third peep!')
+    visit '/'
+
+    peeps = page.all('.peep')
+   
+    expect(peeps[0]).to have_content "Third peep!"
+    expect(peeps[1]).to have_content "Second peep!"
+    expect(peeps[2]).to have_content "First peep!"
+  end
+  
+  #Should I be stubbing time values to see whether?
```

Comment:
You can use Timecop to freeze times - we try to stub non-deterministic values in our tests including time. Timecop's not the only solution, as you could also create stubs instead - although sometimes this requires a bit more of a refactor.